### PR TITLE
fixing-moving-buttons-in-toolbar

### PR DESCRIPTION
**Pull request by tesonep**
The border style of the toolbar button should not have style different than the one when it is in normal state.

Having additional border makes it bigger and it moves, producing a flick.
The SpToolbarMorph should be rigid vertically and spaceFill horizontally to honor the values set has height by the SpToolbarDisplayMode
The extent when the SpToolbarDisplayModeIconAndLabel should be square.

Fix https://github.com/pharo-vcs/iceberg/issues/1431

### CODE REVIEW

**Comment by tinchodias** [comment]
looks good

File: src/Spec2-Core/SpToolbarDisplayModeIconAndLabel.class.st [hunk 1]
```diff
@@ -17,7 +17,7 @@ SpToolbarDisplayModeIconAndLabel >> configureButton: aButton item: aToolbarItem
 
 { #category : #accessing }
 SpToolbarDisplayModeIconAndLabel >> extent [
-	^ 45@40
+	^ 45@45
```
Comment (estebanlm) [inline src/Spec2-Core/SpToolbarDisplayModeIconAndLabel.class.st:20]: I do not like this one. 40 is already very big. Why is this needed?

Comment (tesonep) [reply src/Spec2-Core/SpToolbarDisplayModeIconAndLabel.class.st:20]: To squared. if not we can use 40@40

Comment (estebanlm) [reply src/Spec2-Core/SpToolbarDisplayModeIconAndLabel.class.st:20]: mmm... ok ?
let´s test it.
